app/service/parsing_svc.py

```diff
@@ -1,14 +1,48 @@
-import plugins.stockpile.parsers.standard as parsers
-import plugins.stockpile.parsers.mimikatz as mimikatz_parser
+import glob
 from base64 import b64decode
+from pydoc import locate
 
 from app.service.base_service import BaseService
 
 
 class ParsingService(BaseService):
 
     def __init__(self):
+        self.parsers = dict()
         self.log = self.add_service('parsing_svc', self)
+        
+    async def load_parser(self, p_id, name, parser):
+        """
+        Load the specified parser plugin
+        :param p_id:  the id of the yaml file that defines the parser
+        :param name:  the unique name of the parser
+        :param parser:  an instance of the parser class 
+        :return: None
+        """        
+        if name in self.parsers:
+            self.log.warning('Duplicate parser name detected:  %s:%s' % (p_id, name))
+        else:
+            self.parsers[name] = parser       
+        
+    async def load_parsers(self, directory):
+        """
+        Load all parser plugins
+        :param directory:
+        :return: None
+        """
+        for filename in glob.iglob('%s/*.yml' % directory, recursive=False):
+            for entries in self.strip_yml(filename):
+                parser = locate(entries['class'])
+                if parser:
+                    parser = parser(self.log)
+                else:
+                    self.log.warning('Unable to find parser class %s' % entries['class'])
+                    next
```

Comment:
statement seems to have no effect and can be replaced with a function call or built in (continue)

Reply:
Not sure what you mean, the intent of this clause is to log a warning if the parser class cannot be found.  Not sure what you are recommending, remove the warning log message?

Reply:
here I'm referring to "next" which doesn't do anything

Reply:
next is not a built in and must be called () as a function, so here it isn't doing anything. 

just to try it out:

what you have here
```
for x in range(20):
    if x % 2:
        print('odd', x)
        next
    print('even', x)
```

vs replacing with continue
```
for x in range(20):
    if x % 2:
        print('odd', x)
        continue
    print('even', x)
```